_projects/git_task_list.md

```diff
@@ -0,0 +1,96 @@
+---
+collaborating_projects:
+  - gsoc-prep-tasks
+desc: "List open tasks from various issue tracker. Can be filtered to only show
+  newcomer friendly tasks."
+difficulty: medium
+initiatives:
+  - GSoC
+issues:
+  - https://github.com/coala/projects/issues/480
+  - https://github.com/coala/coala/issues/3615
+  - https://github.com/coala/coala/issues/4080
+markdown: git_task_list.md
+mentors:
+  - sims1253
+  - yukiisbored
+  - yashovardhanagrawal
+name: "Git Task List"
+requirements:
+  - "The applicant should have at least one patch merged to any of the coala repositories."
+  - "Familiarity with Git, GitHub API Gerrit API, Phabricator API, and GitLab API"
+  - "Familiarity with EmberJS, Javascript, and GitHub Pages"
+status: in_progress
+tags:
+  - community
+  - web
+---
+The basic idea is an extended version of [GSOC Task List](http://summerofcode.github.io/gsoc-prep-tasks).
+
+In this project, tasks means open issues and unmerged pull request coming from
+GSOC open source projects.
```

Comment:
I think "open source projects" is better since our target is to make available for a lot of FOSS orgs.